### ISSUE TITLE
Question about train vs validation accuracy

### ISSUE THREAD

**Issue by alexkararo** [espnet/espnet#2583]
Hello, 

I am trying to train an ASR system on Librispeech recipe. 
After 20 epochs, I am curious to why is the validation/main/acc > main/acc: [acc-graph](https://postimg.cc/jWjnKk1x)
The same trend can be observed in the loss graph: [loss-graph](https://postimg.cc/n9qC08Xk)

The loss chart shows that the loss on the train set (main/loss) is larger than the loss on the validation set (validation/main/loss). This is clearly something I was not expecting. In any DNN experiment I made so far the loss on the train set was smaller than the loss on the validation set. Do you have any explanation for this? Am I doing something wrong?

Thank you!

**Comment by kamo-naoyuki** [comment]
Is this transformer model? It looks not converged yet. 10epoch is too small.

**Comment by kan-bayashi** [comment]
Maybe this is caused by SpecAug.
To calculate the validation score, the specaug is not applied.
Therefore, the accuracy will be higher than that of the training data.
In my experiments, after around 50 epochs, the training accuracy became higher than the validation.

**Comment by alexkararo** [comment]
Hello, 

Yes, it is the transformer model. 
Thank you all for the explanation, now it makes sense.
